Source/SpeechToTextV1/WebSockets/SpeechToTextSession.swift

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2016, 2019.
+ * (C) Copyright IBM Corp. 2016, 2020.
```

Comment:
Was 2020 wrong?

I'll stop now with this comment, but you should check all these.